packages/core/core/src/requests/PathRequest.js

```diff
@@ -194,7 +195,14 @@ export class ResolverRunner {
     let invalidateOnFileChange = [];
     let invalidateOnEnvChange = [];
     for (let resolver of resolvers) {
+      let measurement;
       try {
+        measurement = applicationProfiler.createMeasurement(resolver.name, {
```

Comment:
Flagging this one as part of the same series of comments. (not sure if i'm gonna catch em all :pokeball:)

Reply:
I'll just search the code for all the `createMeasurement()` calls and address this.